### DISCUSSION TITLE
Assistance on update of win10widgets

### DISCUSSION THREAD

**Discussion by AlterMind64** [MinhThienDX/win10widgets#50] [Q&A] [answered]
Hi, I'm a real of the slim display of Win10 Widgets but I was also missing the GPU indicator, among others (e.g. I'd like to add more disks displayed), then I found this Git fork, linked from tjmarkham's Git comments.
I'd love to update my initial rainmeter/win10widgets version fetched from win10widgets.com with the release found here but as I'm not a developer I actually need something that looks like an installer/updater. Can you assist me here please or give some pointers ?
Thanks a lot for your feedback !

![Screenshot](https://user-images.githubusercontent.com/88384503/210284245-c720dbe8-edef-4a59-b70d-735bfcdb00f9.png)

**Comment by MinhThienDX** [comment, marked as answer]
Hello @AlterMind64, don't worry.
Would you kindly go to https://github.com/MinhThienDX/win10widgets
Then download the zip file.
Finally extract it into Rainmeter folder inside your Documents folder.

Now you need to know what is the difference between your skin files (old) and the files you just extracted (new).
For that purpose I recommend Beyond Compare https://www.scootersoftware.com/
Just compare folders and you will know exactly what need to be done.

**Reply by AlterMind64** [reply]
Thank you so much **@MinhThienDX**  for the feedback ! Tbh I was madly searching for that 'zip' file location until I discovered it was available from the green "Code" button which was showing a drop-down arrow ^_^ (yeah told you I wasn't familiar with this). Decided to overwrite my whole file structure with the one from the zip file as I didn't customise anything really relevant yet. I also updated Rainmeter (from 2016 to 2022!).
I now also better understand the skins concept, and understand that I'm actually using only one skin : the "Performance combo" to my liking (as shown below, now also displaying the GPU graph).

-> Also could you give me some hints to easily (i.e. via GUI if possible) **show several disks** to the view ? I would need C: F: and M: ideally ?

-> Lastly, I've seen that the 'About' submenu of Rainmeter in the system tray displays an error log in the one of the dialog box tabs : modules are not found/loaded. Are you interested in a screenshot or is this not worth (bc no more actively maintained) ?

![Clipboard02](https://user-images.githubusercontent.com/88384503/211125662-31a8f3fc-dad1-40db-9ebb-02299bd103b0.png)

**Reply by MinhThienDX** [reply]
> -> Also could you give me some hints to easily (i.e. via GUI if possible) **show several disks** to the view ? I would need C: F: and M: ideally ?

You just right click and choose edit skin to see how it works.
Then you can tinker (remember to make backup of the original file) to show other drives.

> -> Lastly, I've seen that the 'About' submenu of Rainmeter in the system tray displays an error log in the one of the dialog box tabs : modules are not found/loaded. Are you interested in a screenshot or is this not worth (bc no more actively maintained) ?

I know what you are talking about so screenshot is not needed.
The log may specify exactly which modules are causing errors, probably dll of other apps.
You can fix them 1 by 1 if you want/need.
But if your skin works without those modules, you can just ignore it.

**Reply by AlterMind64** [reply]
Thanks @MinhThienDX for the hints. I gave it a first look and tbh it seems quite complicated and mixing hard-coded settings with variable settings -- it won't be an easy task but I'm up to it. Once done and if stable, I'll share it with you and you'll evaluate if it's worth sharing.
